core/src/banking_stage/consumer.rs

```diff
@@ -650,6 +655,22 @@ impl Consumer {
             batch.sanitized_transactions().len(),
         );
 
+        for ((result, _nonce), tx) in loaded_transactions
+            .iter()
+            .zip(batch.sanitized_transactions())
+        {
+            if let Err(err) = result {
+                match err {
+                    TransactionError::AccountNotFound
+                    | TransactionError::InsufficientFundsForFee
+                    | TransactionError::InvalidAccountForFee => {
+                        self.invalid_fee_payer_filter.add(*tx.message().fee_payer());
```

Comment:
Here is the only place we add to our filter currently.

These 3 errors:
- `AccountNotFound` - could not find fee-payer during load
- `InsufficientFundsForFee` - fee-payer exists, but doesn't have enough lamports
- `InvalidAccountForFee` - this account can't pay fees

Reply:
~won't `AccountNotFound` get thrown for accounts besides the fee-payer?~

:my_eyes_are_bleeding:
https://github.com/solana-labs/solana/blob/dbe4017143a24fbac0ba07858f7a5c628a81b5e5/accounts-db/src/accounts.rs#L493-L497

Reply:
I think that was the only instance😭
Maybe it's better to swap that error with InsufficientFunds or rename it to FeePayerAccoutNotFound so we don't lose the distinction

Reply:
> I think that was the only instance😭 Maybe it's better to swap that error with InsufficientFunds or rename it to FeePayerAccoutNotFound so we don't lose the distinction

could. separate pr tho